components/StandardListItem/StandardListItemStyles.scss

```diff
@@ -0,0 +1,58 @@
+.StandardListItem {
+
+  display: -webkit-flex;
+  display:         flex;
+  -webkit-align-items: center;
+          align-items: center;
+
+  text-align: center;
+
+  width: 100%;
+
+  // background-color: honeydew;
+  padding: 20px 15px;
```

Comment:
Can you please reduce it to have 15px from top/bottom and 20px from left/right? And further, this should change for the mobile view (<768px) to 12px from top/bottom and 15px from left/right.